### DISCUSSION TITLE
Unexpected behavior when only one atom of a certain type is present in the simulation cell

### DISCUSSION THREAD

**Discussion by Sokseiha** [deepmodeling/deepmd-kit#2051] [Q&A] [answered]
Hello,

I'm running a simulation using a DeePMD potential (v 2.1.0) that I previously trained for polymers containing Li-ions. I noticed that when I have only one Li-ion in my simulation cell, the mean square displacement (MSD) of Li is always zero. On the other hands, if I have more that one Li in the cell, the MSD increases linearly with time as expected. I'm wondering if this is bug in the code but I might be wrong and any suggestions on how to solve this problem is greatly appreciated.
I also attached the link to the input files in case some of you want to reproduce the behavior I just described.
https://drive.google.com/file/d/1YdrnrlC-5vmLdNeBzAImppn7IIZ7Zhcq/view?usp=share_link

Best,
Seiha.

**Comment by njzjz** [comment, marked as answer]
It should be expected as you set [com](https://docs.lammps.org/compute_msd.html) to `yes`.

> If the com option is set to yes then the effect of any drift in the center-of-mass of the group of atoms is subtracted out before the displacement of each atom is calculated.
